app/models/topic.rb

```diff
@@ -126,6 +126,11 @@ def assign(user_id=2, assigned_to)
     self.save
   end
 
+  def self.bulk_assign(post_attributes, assigned_to)
+    Post.create(post_attributes)
```

Comment:
This is great; nice one for using `create` with an array - that's a good tidy up even though I think `create` just loops through the array internally (so still does one query for each insert).